src/goto-programs/safety_checker.h

```diff
@@ -45,4 +45,49 @@ class safety_checkert:public messaget
   const namespacet &ns;
 };
 
+/// \brief The best of two results
+inline safety_checkert::resultt operator||(
+    safety_checkert::resultt const &r1,
+    safety_checkert::resultt const &r2)
+{
+  return r1==safety_checkert::resultt::SAFE?safety_checkert::resultt::SAFE:
```

Comment:
Spaces around `?` and `:`, and `=` below (some of the CBMC codebase has compressed spacing, but thankfully conventional spacing is now allowed :))